api/src/handlers/blocks_api.rs

```diff
@@ -90,12 +90,10 @@ impl HeaderHandler {
 			let oid = get_output_v2(&self.chain, &commit, false, false)?.1;
 			match w(&self.chain)?.get_header_for_output(&oid) {
 				Ok(header) => return Ok(header.hash()),
-				Err(_) => return Err(ErrorKind::NotFound)?,
+				Err(_) => return Err(ErrorKind::NotFound.into()),
 			}
 		}
-		return Err(ErrorKind::Argument(
-			"not a valid hash, height or output commit".to_owned(),
-		))?;
+		Err(ErrorKind::Argument("not a valid hash, height or output commit".to_owned()).into())
```

Comment:
We are using `to_owned()`, `to_string()` and `into()` in different places to convert a `&str` to `String`, for consistency we should just pick one and used it everywhere.